lib/graphql_devise/rails/routes.rb

```diff
@@ -35,9 +34,11 @@ def mount_graphql_devise_for(resource, opts = {})
           new_mutation
         end
 
-        GraphqlDevise::Types::MutationType.field("#{mapping_name}_#{action}", mutation: used_mutation)
+        GraphqlDevise::Types::MutationType.field("#{mapping_name}_#{action.to_s.underscore}", mutation: used_mutation)
```

Comment:
Use snake case everywhere until it is required otherwise

Reply:
Naming here uses snake case, note `"#{mapping_name}_`

Reply:
Yes, calling underscore here is not necessary as you changed line 22